core/server/master/src/test/java/alluxio/master/file/DefaultFileSystemMasterTest.java

```diff
@@ -1570,9 +1571,9 @@ private void startServices() throws Exception {
     mRegistry = new MasterRegistry();
     JournalFactory factory = new MutableJournal.Factory(new URI(mJournalFolder));
     mBlockMaster = new BlockMaster(mRegistry, factory);
-    mExecutorService =
-        Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("FileSystemMasterTest-%d", true));
-    mFileSystemMaster = new FileSystemMaster(mRegistry, factory,
+    mExecutorService = Executors
+        .newFixedThreadPool(2, ThreadFactoryUtils.build("DefaultFileSystemMasterTest-%d", true));
+    mFileSystemMaster = new DefaultFileSystemMaster(mRegistry, factory,
```

Comment:
this init sets a `ExecutorServiceFactory` which is for test purpose. While the standard Factory method does not allow to set this. Do you want to create another factory method in `FileSystemMasterFactory` for this purpose? @jsimsa